### DISCUSSION TITLE
Retrieving all race sessions from multiple seasons

### DISCUSSION THREAD

**Discussion by britnesissom** [theOehrly/Fast-F1#158] [Q&A]
I know you can retrieve multiple race sessions by doing something like this (see below) but is there a way to retrieve all races from more than one season if you don't know the number of races in each one? Sorry if there's an easy way to do it in the documentation; if there is, I missed it.

```python
all_events = dict()
for race in range(1, 10):
    event = ff1.get_session(2021, race, 'R')
    event.load()
    all_events[year] = event
```

**Comment by theOehrly** [comment]
https://theoehrly.github.io/Fast-F1/events.html

The event schedule can list all events from each season.
